scripts/artifacts-building/containers/build-process.sh

```diff
@@ -123,6 +123,10 @@ ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e ro
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=repo_server -v" "install" "config"
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=rsyslog_server -v" "install" "config"
 
+# test one container build contents
```

Comment:
I suggest only doing this if ``PUSH_TO_MIRROR`` != 'yes'`` (include converting its case, just in case).

Also, to reduce the execution time, I'd suggest that the containers not required for this test do not build if  ``PUSH_TO_MIRROR`` != 'yes'``.